src/ensembl/src/header/launchbar/Launchbar.tsx

```diff
@@ -91,10 +91,10 @@ const Launchbar = (props: LaunchbarProps) => {
       <div className={styles.about}>
         <span className={styles.aboutText}>Genome research database</span>
         <LaunchbarButton
-          app="about"
+          app="about-ensembl"
           description="About Ensembl"
           icon={ensemblIcon}
-          enabled={false}
+          enabled={true}
```

Comment:
This should probably be activated only for development (same as entity viewer and custom download buttons)

Reply:
I'm still finding my way around the code - where would I do that?

Reply:
Same file, several lines above (button with `app="entity-viewer"` or `app="custom-download"`). Note what data the `enabled` prop is receiving there.

Reply:
Thanks - done!